### PR TITLE
Feature/v4 support

### PR DESCRIPTION
**Pull request by thepiwo**

### CODE REVIEW

File: migrations/20210608-post-via-burn-added.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+var Sequelize = require('sequelize');
+
+/**
+ * Actions summary:
```
Comment (kenodressel) [inline migrations/20210608-post-via-burn-added.js:6]: should resemble the actual changes

File: utils/basicTippingContractUtil.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ const formatSingleClaim = (contractId, url, [claimGen, amount]) => {
 const formatClaims = returnState => {
   const state = returnState.decodedResult;
 
-  return state.claims ? state.claims.map(([url_id, rawClaim]) => {
+  return state.claims && state.urls ? state.claims.map(([url_id, rawClaim]) => {
```
Comment (kenodressel) [inline utils/basicTippingContractUtil.js:25]: is this change related to V4?

Comment (thepiwo) [reply utils/basicTippingContractUtil.js:25]: I noticed it as issue, but I think it generally could have happened

**Review by kenodressel — COMMENTED** [review]
some tests maybe? :innocent:

**Review by kenodressel — APPROVED** [review]

**Comment by thepiwo** [comment]
should be fine :)
